VertexCompositeProducer/src/ParticleFitter.cc

```diff
@@ -276,8 +276,10 @@ void ParticleFitter::makeCandidates()
     if (!preSelection_(cand)) continue;
     // check if grandaughters are unique
     bool hasDuplicate = false;
+    ParticleSet finalStateColl;
     for (const auto& dau : daughters) {
-      if (dau.hasUserData("daughters") && !isUniqueDaughter(daughters, dau)) { hasDuplicate = true; break; }
+      //if (dau.hasUserData("daughters") && !isUniqueDaughter(daughters, dau)) { hasDuplicate = true; break; }
```

Comment:
I am thinking whether we should make the finalStateColl the same as daughters. In case of LambadaC, we will have finalStateColl having Ks, proton, pi+ and pi-. Do you think we should include Ks or exclude it?